BuildResidentialHPXML/measure.rb

```diff
@@ -2093,23 +2088,37 @@ def arguments(model)
     arg.setDefaultValue(4000)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('pv_system_2_inverter_efficiency', false)
-    arg.setDisplayName('PV System 2: Inverter Efficiency')
-    arg.setUnits('Frac')
-    arg.setDescription('Inverter efficiency of the second PV system.')
-    args << arg
-
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('pv_system_2_system_losses_fraction', false)
-    arg.setDisplayName('PV System 2: System Losses Fraction')
-    arg.setUnits('Frac')
-    arg.setDescription('System losses fraction of the second PV system.')
+    battery_location_choices = OpenStudio::StringVector.new
+    battery_location_choices << Constants.Auto
+    battery_location_choices << 'none'
+    battery_location_choices << HPXML::LocationLivingSpace
+    battery_location_choices << HPXML::LocationBasementConditioned
+    battery_location_choices << HPXML::LocationBasementUnconditioned
+    battery_location_choices << HPXML::LocationCrawlspaceVented
+    battery_location_choices << HPXML::LocationCrawlspaceUnvented
+    battery_location_choices << HPXML::LocationCrawlspaceConditioned
+    battery_location_choices << HPXML::LocationAtticVented
+    battery_location_choices << HPXML::LocationAtticUnvented
+    battery_location_choices << HPXML::LocationGarage
+    battery_location_choices << HPXML::LocationExteriorWall
+    battery_location_choices << HPXML::LocationRoofDeck
+    battery_location_choices << HPXML::LocationOutside
+    battery_location_choices << HPXML::LocationOtherHousingUnit
+    battery_location_choices << HPXML::LocationOtherHeatedSpace
+    battery_location_choices << HPXML::LocationOtherMultifamilyBufferSpace
+    battery_location_choices << HPXML::LocationOtherNonFreezingSpace
```

Comment:
Let's remove everything after LocationGarage other than LocationOutside. (Note that most of them would probably have ended up with simulation errors if you had tested them, since they are not modeled as thermal zones.)